community/csv/src/main/java/org/neo4j/csv/reader/Configuration.java

```diff
@@ -25,6 +25,13 @@
 public interface Configuration
 {
     /**
+     * TODO: Default value is false here, but it is our intention to flip this to true at some point
+     * because of how it better complies with common expectancy of behavior. It may be least disruptive
+     * to do this when changing major version of the product.
+     */
+    boolean DEFAULT_LEGACY_STYLE_QUOTING = true;
```

Comment:
Having read the tests, it looks like legacy-style is the existing neo4j behaviour, and not-legacy is the RFC behaviour. So then I guess only the comment above should say the default is true, but we plan to flip to false.